client/src/members/Members.js

```diff
@@ -51,6 +47,12 @@ class Members extends Component {
           members: json,
         });
       });
+      console.log('Search for new members list attempted')
+      if(Members.members === undefined){
+        console.log('No results were successfully returned')
+      }
+      
+       
```

Comment:
This task requires that such feedback is display to the *user*, not just in the console. Please update this code to conditionally render a text element (or similar) to display to the user when no results have been found. See issue #252.